src/collectors/collector_registry.go

```diff
@@ -10,4 +10,5 @@ var deviceMap = map[string]interface{}{
 	"pfsense":   devices.DevicePfsense{},
 	"procurve":  devices.DeviceProcurve{},
 	"vyatta":    devices.DeviceVyatta{},
+	"mikrotik":  devices.DeviceMikrotik{},
```

Comment:
Move this up to keep these alpha sorted?

Reply:
Done